document/core/exec/instructions.rst

```diff
@@ -192,13 +192,22 @@ Reference Instructions
 
 .. _exec-ref.null:
 
-:math:`\REFNULL~\X{ht}`
+:math:`\REFNULL~x`
 .......................
 
-1. Push the value :math:`\REFNULL~\X{ht}` to the stack.
+1. Let :math:`F` be the :ref:`current <exec-notation-textual>` :ref:`frame <syntax-frame>`.
 
-.. note::
-   No formal reduction rule is required for this instruction, since the |REFNULL| instruction is already a :ref:`value <syntax-val>`.
```

Comment:
I think a variation of this note still applies, since we only reduce _some_ ref.null's.

Reply:
<img width="973" alt="Screenshot 2023-11-14 at 8 13 01 PM" src="https://github.com/WebAssembly/gc/assets/50018375/7cceae35-676f-4655-899d-27e7c86086d2">
I added a note like this. If you think there are better words, feel free to change it.